<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);

QCheckBox:{
	padding: 10px;
}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30px &quot;Fredoka&quot;;
font-weight: 800;
qproperty-alignment: AlignCenter;
color: #BA8F4A</string>
   </property>
   <property name="text">
    <string>LaptopShield+</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #008000;
font-family: &quot;fredoka&quot;;
font-size : 15px;
font-weight: bold;
text-align:center;
float:right
</string>
   </property>
   <property name="text">
    <string>OBJECTS SAVED SUCCESSFULLY</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/OneDrive/Pictures/PROJECT/UG.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>290</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/Downloads/balme4-removebg-preview.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mark_icon">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/raphe/OneDrive/Pictures/PROJECT/completed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Fredoka;
font-weight: bold;
font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Great! You have been cleared for exit Successfully</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Fredoka;
font-weight: bold;
font-size: 15px;</string>
   </property>
   <property name="text">
    <string>Please Rate Us : </string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>97</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QCheckBox" name="checkbox_good">
      <property name="styleSheet">
       <string notr="true">color: #008000;
font-family: &quot;fredoka&quot;;
font-size : 12px;
font-weight: bold;
padding: 0 10px </string>
      </property>
      <property name="text">
       <string>Very Good</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkbox_average">
      <property name="styleSheet">
       <string notr="true">color: #BA8F4A;
font-family: &quot;fredoka&quot;;
font-size : 12px;
font-weight: bold;
padding: 0 10px ;</string>
      </property>
      <property name="text">
       <string>Average</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkbox_bad">
      <property name="styleSheet">
       <string notr="true">color: #000;
font-family: &quot;fredoka&quot;;
font-size : 12px;
padding: 0 10px ;
font-weight: bold;

</string>
      </property>
      <property name="text">
       <string>Not Good</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
